examples/pytorch/test_examples.py

```diff
@@ -195,6 +195,7 @@ def test_run_ner(self):
             --per_device_train_batch_size=2
             --per_device_eval_batch_size=2
             --num_train_epochs={epochs}
+            --seed 7
```

Comment:
This shows, how the new "fast" init leads to different initialization which in some sense breaks backwards compatibility for seed initialization